kcidb/__init__.py

```diff
@@ -250,10 +250,17 @@ def validate_main():
     sys.excepthook = misc.log_and_print_excepthook
     description = 'kcidb-validate - Validate I/O JSON data'
     parser = misc.ArgumentParser(description=description)
```

Comment:
We can simply add the argument right here, after creating the parser.  It would also be good to be clear that we need the schema **version** (and not e.g. a schema file) in the metavar, and describe what the provided schema version would be used for, in the help message. So, something like this:

```suggestion
    parser = misc.ArgumentParser(description=description)
    parser.add_argument(
        'schema_version',
        metavar="SCHEMA_VERSION",
        type=non_negative_int,
        help='Validate against the schema with the specified major version, '
        'instead of the latest one.',
        nargs='?'
    )
```

An improvement could be to explicitly specify the available version numbers to the argument parser via a `choice`, or a version number range via a custom argument `type`. Although the latter would require us validating that our major version numbers are consecutive (e.g. in `kcidb_io.schema.misc.Version.__init__()`). Both of these could wait until we're ready to unify schema version specification.